src/renderers/shaders/ShaderChunk/bsdfs.glsl

```diff
@@ -1,11 +1,19 @@
-float calcLightAttenuation( const in float lightDistance, const in float cutoffDistance, const in float decayExponent ) {
+// convert from intensity (power / solid angle) to irradiance assuming a spherical light falloff
+// this is only valid for spot and point lights
+// physically correct inverse quadratic falloff is achieved when decayExponent is 2.
+// discussed here: https://github.com/mrdoob/three.js/issues/7608
+float intensityToIrradianceSphericalFalloffCoefficient( const in float lightDistance, const in float cutoffDistance, const in float decayExponent ) {
 
 	if ( decayExponent > 0.0 ) {
 
-	  return pow( saturate( -lightDistance / cutoffDistance + 1.0 ), decayExponent );
+		float distanceFalloff = 1.0 / pow( max( lightDistance, EPSILON ), decayExponent );
```

Comment:
> what concerns me is that users who create a scene of molecules, and users who create a galaxy, likely will not have distance units of meters.

Artists can still use constant lights (decay=0) with the range limiting, or they can use linear falloff (decay=1), and both of these are very easy to get what you want even if you are using arbitrary units for distance.  Yes, these lead to non-physical lighting but the alternative you are proposing is also a non-physical lighting scenario.  Thus I believe that we still have these situation covered as well as we did before -- we will still support non-physical but easy to setup lighting scenarios.